src/lint/collect-header-diagnostics.ts

```diff
@@ -0,0 +1,113 @@
+import type { LintingError } from './algorithm-error-reporter-type';
+
+import { getLocation, indexWithinElementToTrueLocation } from './utils';
+
+const ruleId = 'header-format';
+
+export function collectHeaderDiagnostics(
+  dom: any,
+  headers: { element: Element; contents: string }[]
+) {
+  let lintingErrors: LintingError[] = [];
+
+  for (let { element, contents } of headers) {
+    if (!/\(.*\)$/.test(contents) || / Operator \( `[^`]+` \)$/.test(contents)) {
+      continue;
+    }
+
+    let name = contents.substring(0, contents.indexOf('('));
+    let params = contents.substring(contents.indexOf('(') + 1, contents.length - 1);
+
+    if (!/[\S] $/.test(name)) {
+      let { line, column } = indexWithinElementToTrueLocation(
+        getLocation(dom, element),
+        contents,
+        name.length
```

Comment:
```suggestion
        name.length - 1,
```

Reply:
Hm. It feels kind of weird to point to the `e` in `Example( )`, but sure, I guess.

Reply:
It will also point to the second space in <pre><code>Example  ( )</pre></code> though.